test/dummy/config/application.rb

```diff
@@ -33,5 +33,9 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+
+    if Rails::VERSION::STRING.match?(/^7.0/)
```

Comment:
I'm not even sure we need the condition, I think `config` being an `OrderedOptions` would allow writing to it always.

Reply:
[CI complained](https://github.com/Shopify/maintenance_tasks/runs/2945274445?check_suite_focus=true) when setting up the db on previous Rails versions:

![Screen Shot 2021-06-30 at 10 33 08 AM](https://user-images.githubusercontent.com/22918438/123979270-926afa80-d98e-11eb-92b2-a5753063909f.png)

Reply:
Oh ok it's not using ActiveSupport::Configurable. 👍